### PR TITLE
YLP-514 Notifications screen

### PR DESCRIPTION
**Pull request by leovalette**
This PR only implements the UI
Mocked data will be removed in [YLP-461](https://diabeloop.atlassian.net/browse/YLP-461) or [YLP-465](https://diabeloop.atlassian.net/browse/YLP-465)

### CODE REVIEW

File: packages/yourloops/pages/notifications/notifications.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  const notifs: INotification[] = [fakeNotif1, fakeNotif2, fakeNotif3];
+
+  return (
+    <Fragment>
+      <NotificationHeader />
+      <Container maxWidth="lg" style={{ marginTop: "1em" }}>
+        <List>
+          {notifs.sort(sortNotification).map((notification, index) => (
+            <ListItem key={index} style={{ padding: "8px 0" }} divider={index !== notifs.length - 1}>
+              <Notification notification={notification} userRole={user?.roles && user.roles[0]} />
```
Comment (Nas84) [inline packages/yourloops/pages/notifications/notifications.tsx:107]: i would propose to split the notification props in order to remove the duplication.

**Review by Nas84 — CHANGES_REQUESTED** [review]

File: packages/yourloops/pages/notifications/notifications.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+/**
```
[the remark this answers is not in the document]
Comment (leovalette) [reply packages/yourloops/pages/notifications/notifications.tsx:1]: notifications-list sounds good to me

**Review by Nas84 — APPROVED** [review]
lgtm

**Review by franck-fourel — CHANGES_REQUESTED** [review]
A small change requested
Wondering as well if we wanted to have something highlighting the fact that a notification was waiting for an action: the red circle on top of the bell ?

**Pull request closed without merging**